patches/net/minecraft/world/level/block/Block.java.patch

```diff
@@ -11,7 +11,7 @@
      private final Holder.Reference<Block> builtInRegistryHolder = BuiltInRegistries.BLOCK.createIntrusiveHolder(this);
 -    public static final IdMapper<BlockState> BLOCK_STATE_REGISTRY = new IdMapper<>();
 +    @Deprecated //Forge: Do not use, use GameRegistry
-+    public static final IdMapper<BlockState> BLOCK_STATE_REGISTRY = net.neoforged.neoforge.registries.GameData.getBlockStateIDMap();
++    public static final IdMapper<BlockState> BLOCK_STATE_REGISTRY = net.neoforged.neoforge.registries.RegistryManager.getBlockStateIdMap();
```

Comment:
Just checking to make sure this was the replacement to `GameData.getBlockStateIDMap()`, while it is pretty niche, I do need a full list of blockstates on load to handle an odd recipe presenter in something like JEI. It LOOKS to be the replacement, just wanted to be sure.

(sleep deprived cursory glance tells me it is just moving to RegistryManager, and this is an accessor, so was perhaps a bit confused as to why have both. No qualms with them both existing, just wanted clarifications :) )

Reply:
It is indeed just moving to RegistryManager